issue  #974 -Images stretched out on `/teachers` & issue #973 -Footer not sticking to the bottom of the page of small pages

abstrekt opened the pull request:
Fixes #
[issue  #974 Images stretched out on `/teachers`](https://github.com/CircuitVerse/CircuitVerse/issues/974)
[issue #973 Footer not sticking to the bottom of the page of small pages](https://github.com/CircuitVerse/CircuitVerse/issues/973)
I hope it fixes the issue

tachyons commented on app/assets/stylesheets/logixRelease.scss:
```diff
@@ -271,6 +271,10 @@ a.btn.btn-outline-danger {
   padding-top: 40px;
 }
 
+.col-lg-6{
```
Do not override core bootstrap classes

abstrekt replied:
shall I add new class then? because there are only bootstrap classes in that div

abstrekt replied:
okay so i have applied ```  align-items: center;``` to the class ```.row-about```  instead of the bootstrap classes which targets these elements.

aayush-05 replied:
Don't change the row-about class, it is getting used at a lot of places which may not go well with this change. Just add a custom class to the element you want to change and then add the styles to it. @abstrekt

abstrekt replied:
ok got it @aayush-05

abstrekt replied:
please review, I added a custom class ```.benefits``` wherever required. @aayush-05

thiefie commented:
Please add a screenshot of the changes. Also link the issue in the PR body.

nitin10s commented:
The images would be replaced by those of new UI

nitin10s commented:
I'll send them to you actually by tomorrow, maybe even replace the old ones in this PR.

abstrekt commented:
@nitin10s okay please do send the new images, I was working on it now actually, the height was defined in the html of the first two images, thus it stretched as it's aspect ratio didn't fit!
So I removed them & let the first two images auto adjust, I guess if those images are replaced by something with similar aspect ratio the problem would persist if the height styling stayed in the html, this is what I have done: 
![TEACHERS benefit](https://user-images.githubusercontent.com/42478217/75678861-8e549d00-5cb4-11ea-94cf-d7a205a91b57.png)
also increased the height of last two images to 250px so that they're readable from a screen.
Will this work?

abstrekt commented:
[issue#974](https://github.com/CircuitVerse/CircuitVerse/issues/974)
* updated the teacher-1 about with the new images
* adjusted its size to make it readable
* fixed stretching of images

please review @nitin10s @tachyons @thiefie

abstrekt commented:
## screenshot
![teachers-about issue#974 ](https://user-images.githubusercontent.com/42478217/75755476-b693d880-5d54-11ea-965b-ec0015659ed2.png)

abstrekt commented:
> 
> 
> Please add a screenshot of the changes. Also link the issue in the PR body.

updated!

abstrekt commented:
## screenshot footer issue #973 
![issue footer](https://user-images.githubusercontent.com/42478217/75765157-10040380-5d65-11ea-81f2-bbcb44a0bb95.png)

nitin10s commented:
For some reason, I am not able to review this PR locally. Is there any issue with the PR?

abstrekt commented:
> 
> 
> The UI looks good. Although I'll suggest you could give better names to classes for better maintenance. Also should we not have separate PR for footer? @tachyons

what class name do you suggest? I will update it right now! , there's an ID ```teachers-benefits```  somewhere I guess thus to avoid confusions I only named this class benefits.

nitin10s commented on app/assets/stylesheets/logixRelease.scss:
```diff
@@ -271,6 +271,10 @@ a.btn.btn-outline-danger {
   padding-top: 40px;
 }
 
+.benefits{
```
You can name it align-center

abstrekt replied:
> 
> 
> You can name it align-center

done!

aayush-05 commented:
The fix you did for the footer is more like a hotfix. It would be creating a lot of space on other pages. We need a way to remove that empty div altogether. May be the position attribute can help.
![image](https://user-images.githubusercontent.com/47032027/75773179-65471180-5d73-11ea-9341-b913acbaa17a.png)
Also, what are these two files
![image](https://user-images.githubusercontent.com/47032027/75773240-80198600-5d73-11ea-8a6b-35ce396d5d5b.png)

abstrekt commented:
> 
> 
> The fix you did for the footer is more like a hotfix. It would be creating a lot of space on other pages. We need a way to remove that empty div altogether. May be the position attribute can help.
> ![image](https://user-images.githubusercontent.com/47032027/75773179-65471180-5d73-11ea-9341-b913acbaa17a.png)
> Also, what are these two files
> ![image](https://user-images.githubusercontent.com/47032027/75773240-80198600-5d73-11ea-8a6b-35ce396d5d5b.png)

Ok I'll look into it now, these two files were the footer logo, I had fork syncing problem, will remove the unwanted image.

nitin10s commented:
Flexbox is used for such scenarios, as far as I know.

abstrekt commented:
## screenshot
![footer fix](https://user-images.githubusercontent.com/42478217/75774830-cc19fa00-5d76-11ea-80cb-c4a6cd999d45.png)

please check @nitin10s @aayush-05

abstrekt commented on app/assets/stylesheets/layout.scss:
```diff
@@ -76,9 +76,10 @@
-  left: 40%;
-  position: absolute;
-  top: 35%;
+  display: flex;
```
Ok mr Bot!

abstrekt commented:
> 
> 
> > The fix you did for the footer is more like a hotfix. It would be creating a lot of space on other pages. We need a way to remove that empty div altogether. May be the position attribute can help.
> > ![image](https://user-images.githubusercontent.com/47032027/75773179-65471180-5d73-11ea-9341-b913acbaa17a.png)
> > Also, what are these two files
> > ![image](https://user-images.githubusercontent.com/47032027/75773240-80198600-5d73-11ea-8a6b-35ce396d5d5b.png)
> 
> Ok I'll look into it now, these two files were the footer logo, I had fork syncing problem, will remove the unwanted image.

I think that empty div is playing some role in having some spaces between the contents & the footer that makes it look good structurally, removing it will need a lot of resizing for the pages it effects I guess?

abstrekt commented:
> 
> 
> For some reason, I am not able to review this PR locally. Is there any issue with the PR?

I'm quit not sure what might be the problem!

aayush-05 commented:
Looks good to me now.

tachyons commented on app/assets/stylesheets/logixRelease.scss:
```diff
@@ -271,6 +271,10 @@ a.btn.btn-outline-danger {
   padding-top: 40px;
 }
 
+.align-center{
```
`Opening curly brace `{` should be preceded by one space `

abstrekt replied:
Done!

nitin10s commented:
@aayush-05
